templates/studygroups/email/feedback-submitted.html

```diff
@@ -1,7 +1,7 @@
 {% extends 'email_base.html' %}
 {% load i18n %}
 {% block body %}
-<p>{% trans "Hi there" %}</p>
+<p>{% trans "Hi {{learner_name}}" %}</p>
```

Comment:
@s-kennedy This template is sent to the organizer of a team when a facilitator submits weekly feedback. Did you maybe want to update a different template?